replayer/src/main/java/dk/dbc/service/performance/replayer/ReplayerTask.java

```diff
@@ -43,15 +46,18 @@ public ReplayerTask(Config config, LogCollector logCollector, CallTimeWathcer wa
         this.logLine = logLine;
         this.jobListener = jobListener;
         this.logEntry = logEntry;
+        this.dryRun = config.isDryRun();
     }
 
     @Override
     public void run() {
-        log.debug( "Running: logLine=" + logLine);
-        long duration = sender.send(logLine, logEntry);
+        log.debug("Running: logLine={}{}", logLine, dryRun ? " (dry-run)" : "");
+        long duration = 0;
+        if (!dryRun)
```

Comment:
I'm a really big fan of using curly braces, even if there is only one line in an if-branch...